runbotics/runbotics-orchestrator-ui/src/src-app/views/auth/RegisterPage/Register.page.tsx

```diff
@@ -302,11 +304,11 @@ const RegisterPage: FC = () => {
                     <RouterLink href="/">
                         <Logo height={100} />
                     </RouterLink>
-                    <Typography  
-                        variant="h4"
-                    >
-                        {invitingTenant}
-                    </Typography>
+                    {inviteCodeURL && invitingTenant && (
```

Comment:
inviteCodeURL && invitingTenant should be a boolean variable explaining what is the condition, e.g. shouldShowTenant